### PR TITLE
Use bosh V2 commands in readme and simplify manifest files.

### PR DESCRIPTION
**Pull request by krishnasrinivas**
dkalinin on #bosh slack channel suggested to use style followed in https://github.com/shinji62/minio-dist-boshrelease

### CODE REVIEW

File: manifests/manifest-fs.yml [hunk 1]
```diff
@@ -1,54 +1,39 @@
...
+  instances: 1
+  jobs:
+  - name: minio-server
+    release: minio
+    templates:
+    - name: minio-server
+    properties:
+      credential:
+        accesskey: minio
+	secretkey: minio123
```
Comment (harshavardhana) [inline manifests/manifest-fs.yml]: Indent is wrong here for yaml?

Comment (krishnasrinivas) [reply manifests/manifest-fs.yml]: there was tab instead of spaces, fixed

**Comment by dkalinin** [comment]
Sorry, but I did not suggest anything. And I don't know why I am in this PR.

**Comment by krishnasrinivas** [comment]
Sorry @dkalinin it was a different dkalinin on https://cloudfoundry.slack.com who suggested it

**Pull request merged**